pages/linux/a2disconf.md

```diff
@@ -0,0 +1,12 @@
+# a2disconf
+
+> Disable an Apache configuration file on Debian-based OSes.
+> More information: <https://httpd.apache.org/>.
+
+- Disable a configuration file:
+
+`sudo a2disconf {{configuration_file}}`
```

Comment:
Perhaps this will read better?
```suggestion
`sudo a2disconf {{path/to/configuration_file}}`
```

....same for the all other instances.

Reply:
This argument is not a path, but only a filename. It will then look for a file with that name in `/etc/apache2/conf-available/`.

Reply:
Ah, I see. Let's leave it as-is then